src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/UpToDateCheckImplicitConfiguredInput.cs

```diff
@@ -71,6 +71,14 @@ public static UpToDateCheckImplicitConfiguredInput CreateDisabled(ProjectConfigu
         /// </summary>
         public string? NewestImportInput { get; }
 
+        /// <summary>
+        /// Gets whether the fast up-to-date check has been disabled via the <c>DisableFastUpToDateCheck</c>
+        /// property.
+        /// </summary>
+        /// <remarks>
+        /// When <see langword="true"/>, other properties on this snapshot will not be used, and so
+        /// are not populated.
```

Comment:
What is our criteria for line breaks in code comments? Our code itself is wider than this in our codebase in many places. I've seen various styles throughout for comments. Just looking for a policy that encourages consistency.

Reply:
I'm not sure such a policy exists.